### DISCUSSION TITLE
Bricked? (Resolved)

### DISCUSSION THREAD

**Discussion by rira2005** [pybricks/support#1309] [Q&A]
Hello 
Upgraded a Xiaomi BT TEmp/Hygro
SW: was 1.1.xxx  not the new buggy one)

Flashing worked fine.... afte reboot of the device.... not change to connect...
Any Idea what to do?

08:37:05: Searching for devices
08:37:32: Get Advertising MAC (Web Experimental Features!)
08:37:32: Connecting to: ATC_2C8BF8
08:37:41: MAC: A4C1382C8BF8, Bat: 100%, Vbat: 3075 mV , Temp: 23.87°C, Humi: 40.98%, Count: 3, Flg: 4
08:37:57: Hardware Revision String: B1.6
08:37:57: Software Revision String: V4.5
08:37:57: Firmware Revision String: github.com/pvvx
08:37:57: Detected Mi device
08:37:57: NotSupportedError: GATT operation failed for unknown reason.
08:37:57: Reconnect 1 from 5
08:37:57: Hardware Revision String: B1.6
08:37:57: Software Revision String: V4.5
08:37:57: Firmware Revision String: github.com/pvvx
08:37:57: Detected Mi device
08:37:58: NotSupportedError: GATT operation failed for unknown reason.
08:37:58: Reconnect 2 from 5
08:37:58: Hardware Revision String: B1.6
08:37:58: Software Revision String: V4.5
08:37:58: Firmware Revision String: github.com/pvvx
08:37:58: Detected Mi device
08:37:59: NotSupportedError: GATT operation failed for unknown reason.
08:37:59: Reconnect 3 from 5
08:37:59: Hardware Revision String: B1.6
08:37:59: Software Revision String: V4.5
08:37:59: Firmware Revision String: github.com/pvvx
08:37:59: Detected Mi device
08:38:00: NotSupportedError: GATT operation failed for unknown reason.
08:38:00: Reconnect 4 from 5
08:38:00: Hardware Revision String: B1.6
08:38:00: Software Revision String: V4.5
08:38:00: Firmware Revision String: github.com/pvvx
08:38:00: Detected Mi device
08:38:00: NotSupportedError: GATT operation failed for unknown reason.
08:38:00: Reconnect 5 from 5
08:38:01: Hardware Revision String: B1.6
08:38:01: Software Revision String: V4.5
08:38:01: Firmware Revision String: github.com/pvvx
08:38:01: Detected Mi device
08:38:01: NotSupportedError: GATT operation failed for unknown reason.
08:38:01: Something went wrong, too many reconnects

TIa
Raphael

**Comment by rira2005** [comment]
i am sorry after restarting my pc it worked...

**Reply by laurensvalk** [reply]
No problem. Glad to hear it worked!

**Comment by rira2005** [comment]
remember : use your brain first!
